Manual link decorators for images will not crash decorators for linked text (#8119)
ckeditor/ckeditor5: tests/manual/all-features.md

**Code blocks in the table** - in the table (4x3), in the second and fourth columns should be visible code snippets.

**Horizontal line** - There is the `<hr>` in the source HTML. It should be displayed in the editor.

> **Link images + Link decorators** - in the table (3x2), there be a link and two linked images that uses the manual decorators feature:
> 
>   - Left column: a plain text with two decorators enabled: `Open in a new tab`, `Downloadable`
>   - Middle column: an image with the caption that is a `Gallery link`
>   - Right column: an image without the caption with enabled all decorators (listed above)

Author: ```suggestion
**Link images + Link decorators** - in the table (3x2), there are a linked text and two linked images that uses the manual decorators feature:
```

  - Left column: the text with two decorators enabled: `Open in a new tab`, `Downloadable`
  - Middle column: an image with the caption that is a `Gallery link`
  - Right column: an image without the caption with enabled all decorators (listed above)
```
